vignettes/tips_and_tricks.qmd

```diff
@@ -0,0 +1,31 @@
+---
+title: "Tips & Tricks "
+format:
+  html:
+    css: custom.css
+tbl-colwidths: [20,20,50,10]
+---
+
+This page collects a miscellaneous tips and tricks for **tinyplot**.
+
+# Legend
+
+## Opacity (`alpha`)
+
+By default, the legend inherits the opacity of the plotted elements. In some cases, this may be undesirable. For example, in the following plot, the legend is too light:
+
+```{r}
+library(tinyplot)
+x <- rnorm(1e6)
+y <- x + rnorm(1e6)
+z <- sample(c("a", "b"), 1e6, replace = TRUE)
+dat <- data.frame(x, y, z)
+tinyplot(y ~ x | z, data = dat, alpha = .1, pch = 19)
+```
+
+One solution is to draw our plot in two steps. First, we draw an empty plot with desired legend. Second, we add the actual points on top of the existing canvas.
```

Comment:
Suggested rephrasing (and typo fix)

"One solution is to draw our plot in two steps. First, we draw an empty plot with the desired (zero transparency) legend. Second, we add the transparent points on top of the existing canvas."